src/os/ui/layer/vectorstylecontrols.js

```diff
@@ -112,6 +117,22 @@ os.ui.layer.VectorStyleControlsCtrl = function($scope, $element) {
    */
   this.scope['lineDashOption'] = name ? name : this.scope['lineDashOptions'][1];
 
+  /**
+   * The chosen mappings
+   * @type {Object}
+   */
+  this['ellipseMapping'] = undefined;
+
+  this['allowEllipseConfig'] = this.scope['allowEllipseConfig'] || false;
+
+  /**
+   * Help text for Ellipse configuration
+   * @type {string}
+   */
+  this['configEllipse'] = `This layer does not have columns that support Ellipse styling (Radius, or Semi Minor, Semi 
+    Major, and Orientation). To configure which columns should be mapped be used instead, use the settings cog to
```

Comment:
Wording is a little off here, and perhaps more verbose than needed. Maybe simplify to something like:
> Required ellipse columns are not configured on this layer. To display ellipses, click the settings cog and select the requested columns.